include/deal.II/fe/mapping_fe.h

```diff
@@ -604,6 +607,18 @@ MappingFE<dim, spacedim>::preserves_vertex_locations() const
   return true;
 }
 
+
+template <int dim, int spacedim>
+bool
+MappingFE<dim, spacedim>::is_compatible_with(
+  const ReferenceCell::Type &cell_type) const
```

Comment:
Maybe move to the `.cc` file. This function is generally called via the `virtual` base class function, so the compiler can't inline anyway.